examples/spark2/pom.xml

```diff
@@ -49,6 +49,11 @@
       <artifactId>carbondata-store-sdk</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.carbondata</groupId>
+      <artifactId>carbondata-mv-core</artifactId>
```

Comment:
This is adding always or in case of `mv` profile is selected?

Reply:
Yes, profile added